packages/test/end-to-end-tests/src/test/batching.spec.ts

```diff
@@ -500,5 +500,5 @@ const tests = (args: ITestObjectProvider) => {
 };
 
 describe("Batching", () => {
-    generateTest(tests, { tinylicious: process.argv.includes("--tinylicious") });
+    generateTest(tests, { tinylicious: true });
```

Comment:
this will break our current f5 debugging, and we loose coverage for local-test server. i think i'd prefer if we run both configurations, rather than have to choose

Reply:
i'd be more ok with this if tinylicious was part of server packages, as that would at least ensure we are testing the latest server code, as it is tinylicious always lags

Reply:
i also wonder if our jest tests are a better candidate for tinylicious than our unit/functional tests

Reply:
The flag means that we will generate test targeting tinylious as well.  So I don't think it will lose coverage.
I think it is a feature that we test with older server code here.   It is at least should be backward compatible and only updated explicitly as needed.

Reply:
Not sure why f5 debugging would stop working.  You need to start the tinylicious server if you are going to run the test.

Reply:
for tinylicious it also ends up being on dev versions quite a bit. i like the idea of testing with older versions of tinylicious, just wish it was explicit, rather than implicit. right now, some can easily modify tinylicious, and break out test in packages silently